### PR TITLE
Reader: Add support for reddit embeds

### LINKED ISSUE #1549: Reader: Add support for Reddit embeds
Reddit has a really nice WordPress.com blog—one of my favorites to read—but unfortunately Reader doesn't know what to do with the Reddit embeds that they use a lot: https://upvoted.com/2015/12/14/redditors-photoshop-ewan-mcgregor-in-front-of-a-green-screen/

In Reader, these embeds don't alway include the relevant information, but do link to the Reddit post: https://wordpress.com/read/post/feed/37856615/886109663

screen shot 2015-12-14 at 11 30 15 am

I'd be great if we can make them look more like they should:

screen shot 2015-12-14 at 11 30 22 am

### PR DESCRIPTION
**Pull request by blowery**
http://calypso.localhost:3000/read/feeds/25320833/posts/1454453490 vs https://wordpress.com/read/feeds/25320833/posts/1454453490

Fixes #1549

### CODE REVIEW

File: client/components/embed-container/index.jsx [hunk 1]
```diff
@@ -1,18 +1,14 @@
 /**
- * A component that notices when the content has embeds that require outside JS. Load the outside JS and process the embeds
```
Comment (samouri) [inline client/components/embed-container/index.jsx]: did you mean to delete this comment

Comment (samouri) [inline client/components/embed-container/index.jsx]: nevermind...it just got bounced down

**Comment by shaunandrews** [comment]
Super awesome. I do notice that sometimes the embed isn't loaded on page load — it "flashes" in to place a few seconds after it appears on screen. Not a major problem though.

**Comment by samouri** [comment]
LGTM. 

> Super awesome. I do notice that sometimes the embed isn't loaded on page load — it "flashes" in to place a few seconds after it appears on screen. Not a major problem though.

You are right and its less than ideal.  It would be pretty difficult to fix that though

**Comment by blowery** [comment]
@shaunandrews yeah, not much we can do about that one. Just how reddit embeds work.

**Pull request merged**